### ISSUE TITLE
x, y - numbers -- only if x and y appear together?

### ISSUE THREAD

**Issue by seancorfield** [bbatsov/clojure-style-guide#239]
It seems fairly common to see `x & xs` in destructuring where `x` is some arbitrary "thing" and `xs` is a sequence of such "things".

Yet the guide suggests `x` and `y` are typically numbers.

Perhaps we can refine this: `x` and `y` are likely to be numbers when both are present, such as `[x y]` when a coordinate system is implied, but `x` standing alone tends to mean a arbitrary thing in a lot of cases, and especially when used with `xs`.

I don't have a specific suggestion to improve the wording but wanted to bounce this off the editorial team to see if we can come up with something better here?

**Comment by bbatsov** [comment]
@seancorfield I totally agree. I had originally borrowed this from https://clojure.org/dev/contrib_howto#_coding_guidelines and I didn't bother to extend the explanations, but I think this would be beneficial.
